functest/new_api_test.go

```diff
@@ -50,22 +50,26 @@ func TestBadSeed(t *testing.T) {
 	ctx := context.TODO()
 	rootCfg, err := requester.CreateUserConfig(root.ref, root.privKey)
 	require.NoError(t, err)
-	res, err := requester.SendWithSeed(ctx, TestCallUrl, rootCfg, &requester.RequestConfigJSON{
-		Method: "CreateMember",
-		Params: nil,
-	}, []byte("111"))
+	res, err := requester.SendWithSeed(ctx, TestCallUrl, rootCfg, &requester.Request{
+		JSONRPC: "2.0",
+		ID:      1,
+		Method:  "call.api",
+		Params:  requester.Params{CallSite: "contract.createMember"},
+	}, "111")
 	require.NoError(t, err)
-	require.EqualError(t, contractError(res), "[ checkSeed ] Bad seed param")
+	require.EqualError(t, contractError(res), "[ checkSeed ] Decode error")
```

Comment:
This test doesn't check "BadSeed" anymore. The best solution is to add test for checking new "Decode error" error and change this one input data for getting "Bad seed param" error.